latex/epoch.tex

```diff
@@ -1326,87 +1262,138 @@ \subsection{Reward Transition}
   \label{fig:fund-preservation}
 \end{figure}
 
-Figure~\ref{fig:rules:reward} defines the reward transition rule.
-The reward transition has no preconditions.
+Figure~\ref{fig:defs:reward-update} defines a reward update.
+It consists of four pots:
+\begin{itemize}
+  \item The change to the treasury. This will be a positive value.
+  \item The change to the reserves. This will be a negative value.
+  \item The net reward pot.
+  \item The map of new individual rewards (to be added to the existing rewards).
+  \item The change to the fee pot. This will be a negative value.
+\end{itemize}
+
+%%
+%% Figure - Reward Update Defs
+%%
+\begin{figure}[htb]
+  \emph{Reward Update}
+  \begin{equation*}
+    \RewardUpdate =
+    \left(
+      \begin{array}{r@{~\in~}ll}
+        \Delta t & \Coin & \text{change to the treasury} \\
+        \Delta r & \Coin & \text{change to the reserves} \\
+        \var{rp} & \Coin & \text{new reward pot} \\
+        \var{rs} & \AddrRWD\mapsto\Coin & \text{new individual rewards} \\
+        \Delta f & \Coin & \text{change to the fee pot} \\
+      \end{array}
+    \right)
+  \end{equation*}
+  %
+  \caption{Rewards Update type}
+  \label{fig:defs:reward-update}
+\end{figure}
 
+\clearpage
+
+Figure~\ref{fig:functions:reward-update-creation} defines two functions,
+$\fun{createRUp}$ to create a reward update, and $\fun{applyRUpd}$ to apply a reward update
+to an instance of $\EpochState$.
+
+The $\fun{createRUp}$ function does the following:
 \begin{itemize}
-  \item First we calculate $\var{totalPot}$, the total amount of coin avail for rewards this epoch,
-    , as described in section 6.4 of \cite{delegation_design}. It consists of four pots:
+  \item First we calculate the change to the reserves,
+    as determined by the $\rho$ protocol parameter.
+  \item Next we calculate $\var{totalPot}$, the total amount of coin available for rewards this
+    epoch, as described in section 6.4 of \cite{delegation_design}. It consists of four pots:
     \begin{itemize}
       \item The fee pot, containing the transaction fees from the epoch.
       \item The amount of coin in the deposit pot that is no longer needed, due to decay.
       \item The reward pot, which is the left-over rewards from the previous epoch.
-      \item Some amount of monetary expansion from the reserves, as determined by the
-        $\rho$ protocol parameter.
+      \item The amount of monetary expansion from the reserves, calculated above.
     \end{itemize}
     Note that the fee pot and the decayed amount are taken from the snapshot taken at the
     epoch boundary.  (See~Figure\ref{fig:rules:snapshot}).
-  \item Some proportion of the total pot is moved to the treasury,
+  \item Next we calculate the proportion of the total pot that moved to the treasury,
```

Comment:
The phrasing here is a little weird - we calculate the proportion _that_ moved? So it has already been added to the treasury? Otherwise, maybe "total pot _to be_ moved"?

Reply:
yea, you are right. I was finding it hard to get the tense correct on the creation of a diff that will be applied later. I'll have another go at it!